athenahealth/generic.go

```diff
@@ -9,3 +9,5 @@ type ErrorMessageResponse struct {
 	Message string `json:"errormessage"`
 	Success bool   `json:"success"`
 }
+
+const SuccessfulResponse = "1"
```

Comment:
Let's keep this scoped to the appointments file & athenahealth package for now as we don't have use cases or validation for using it generically yet. Probably doesn't need to be exported either, a var in `appointments.go` should be fine